modules/rosa-autorepair-cli.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+// * nodes/rosa-managing-worker-nodes.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-autorepair-cli_{context}"]
+= Configuring machine pool AutoRepair using the ROSA CLI
+
+You can configure machine pool AutoRepair for your {product-title} cluster by using the ROSA CLI.
+
+
+.Prerequisites
+
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`) on your workstation.
+* You logged in to your Red{nbsp}Hat account by using the `rosa` CLI.
+* You created a {hcp-title} cluster.
+* You have an existing machine pool.
+
+.Procedure
+
+. List the machine pools in the cluster by running the following command:
++
+[source,terminal]
+----
+$ rosa list machinepools --cluster=<cluster_name> 
+----
++
+.Example output
+[source,terminal]
+----
+ID           AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS    TAINTS    AVAILABILITY ZONE  SUBNET                    VERSION  AUTOREPAIR  
+workers      No           2/2       m5.xlarge                          us-east-2a         subnet-0df2ec3377847164f  4.16.6   Yes         
+db-nodes-mp  No           2/2       m5.xlarge                          us-east-2a         subnet-0df2ec3377847164f  4.16.6   Yes  
+----
+
+. Enable or disable AutoRepair on a machine pool:
+
+* To enable or disable AutoRepair for a machine pool, run the following command:
++
+[source,terminal]
+----
+$ rosa edit machinepool --cluster=mycluster --machinepool=<machinepool_name>  --autorepair false
+----
++
+.Example output
+[source,terminal]
+----
+I: Updated machine pool 'machinepool_name' on cluster 'mycluster'
+----
+
+
+.Verification
+
+. Describe the details of the machine pool with the new taints:
++
+[source,terminal]
+----
+$ rosa describe machinepool --cluster=<cluster_name> --machinepool=<machinepool_name>
+----
++
+.Example output
+[source,terminal]
+----
+ID:                            machinepool_name
+Cluster ID:                    <ID_of_cluster>
+Autoscaling:                   No
+Desired replicas:              2
+Current replicas:              2
+Instance type:                 m5.xlarge
+Labels:                        
+Tags:                              
+Taints:                        key1=value1:NoSchedule, key2=value2:NoExecute        
+Availability zone:             us-east-2a
+...
+----
+
+. Verify that the AutoRepair setting is correct for your machine pool in the output.
```

Comment:
Which item in the output is the _AutoRepair setting_?

Reply:
oops, cut it off. Updated